nservicebus/azure-servicebus/topologies/index.md

```diff
@@ -50,7 +49,7 @@ This topology has several drawbacks:
 
 ### Forwarding Topology
 
-`ForwardingTopology` is the new topology that was designed to take advantage of several native broker features offered by Azure Service Bus. Unlike `EndpointOrientedTopology`, it doesn't work with a single topic per publisher. All publishers use a topic bundle (LINK link-to-bundles). Subscriptions  are created under topic bundle with one subscription entity per subscribing endpoint. Each subscription contains multiple rules. A rule per event type that subscribing endpoint is interested in. This allows complete decoupling between publishers and subscribers. All messages received by subscription are forwarded to the input queue of the subscriber.
+`ForwardingTopology` is the new topology that was designed to take advantage of several native broker features offered by Azure Service Bus. Unlike `EndpointOrientedTopology`, it doesn't work with a single topic per publisher. All publishers use a topic bundle. Subscriptions are created under topic bundle with one subscription entity per subscribing endpoint. Each subscription contains multiple rules. A rule per event type that subscribing endpoint is interested in. This allows complete decoupling between publishers and subscribers. All messages received by subscription are forwarded to the input queue of the subscriber.
```

Comment:
We have a task to review all content and fix linking at the end of getting ASB ready, so +1 for that.